src/sentry/models/groupredirect.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+from sentry.db.models import BoundedBigIntegerField, Model, FlexibleForeignKey
+
+
+class GroupRedirect(Model):
+    """
+    Maintains a reference from a group that has been merged (and subsequently
+    deleted) to the group that superceded it.
+    """
+    __core__ = False
+
+    group = FlexibleForeignKey('sentry.Group')
+    previous_group_id = BoundedBigIntegerField(db_index=True, unique=True)
```

Comment:
I considered adding a `timestamp` field here, but wasn't sure it was necessary so opted to avoid it.